test/test_files/StyleLexers/styleLexJSON/sourcetreeconfig.json

```diff
@@ -1,47 +1,47 @@
 {
-  "$id": "13",
+  "$id": "0",
   "AutoRefresh": true,
   "AutoRefreshRemotes": true,
   "CommitTextLinks": null,
   "DraftCommitMsg": "",
   "IncomingSavedCount": 0,
-  "LastCheckedRemotes": "2019-07-12T23:08:32.1713054+02:00",
-  "LastUsedView": 1,
+  "LastCheckedRemotes": "2099-01-01T00:00:00.0+0:00",
```

Comment:
2099?

Reply:
@leeoniya,
The title of the PR #1575 is"chg Anonymizing **test file** in styleLexJSON  😉 
I knew it, but don't worry, this is just **a test file for styleLexJSON**, without any impact on Notepad3.